tests/tests.bats

```diff
@@ -38,15 +38,26 @@ teardown()
 }
 
 @test "tarball_topologies_only 2.2.1" {
-    test_tarball_topologies_only v2.2.x v2.2.1
+    local ver=v2.2.1
+    test_init
+    get_release "$ver"/sof-tplg-"$ver".tar.gz
+    # These should have never been there
+    # rm "$EXTR_REFS"/sof-tplg-v2.2.1/cavs-*.tplg
```

Comment:
@mengdonglin this removal line makes the tests pass in https://github.com/thesofproject/sof-bin/actions/runs/3061834845/jobs/4942097194 /
- #98 

(TESTED)

Reply:
Thank you @marc-hb!